backend/page_ops.py

```
@@ -427,24 +384,168 @@ def delete_branch_page(
+    Convert database row to page dict.
+    
+    Handles datetime serialization and metadata parsing.
+    """
+    if not row:
+        return None
+    
+    page = dict(row)
+    
+    # Convert datetime objects to ISO strings
+    for key in ("created_at", "updated_at"):
+        if key in page and page[key]:
```

Ruff (0.14.10)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)